test/integration/package_int_test.go

```diff
@@ -512,14 +512,14 @@ func (suite *PackageIntegrationTestSuite) TestNormalize() {
 	cp.ExpectExitCode(0)
 
 	cp = ts.SpawnWithOpts(
-		e2e.OptArgs("install", "Charset_normalizer"),
+		e2e.OptArgs("install", "Charset-normalizer"),
```

Comment:
See https://activestatef.atlassian.net/browse/DX-2805?focusedCommentId=53790 for an explanation of why this changed.